networkx/generators/classic.py

```diff
@@ -489,29 +489,40 @@ def cycle_graph(n, create_using=None):
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def dorogovtsev_goltsev_mendes_graph(n, create_using=None):
-    """Returns the hierarchically constructed Dorogovtsev-Goltsev-Mendes graph.
+    r"""Returns the hierarchically constructed Dorogovtsev--Goltsev--Mendes graph.
 
-    The Dorogovtsev-Goltsev-Mendes [1]_ procedure produces a scale-free graph
-    deterministically with the following properties for a given `n`:
-    - Total number of nodes = ``3 * (3**n + 1) / 2``
-    - Total number of edges = ``3 ** (n + 1)``
+    The Dorogovtsev--Goltsev--Mendes [1]_ procedure deterministically produces a
+    scale-free graph with $\frac{3}{2} (3^{n-1} + 1)$ vertices
+    and $3^n$ edges for a given $n$.
+
+    Note that $n$ denotes the number of times the state transition is applied,
+    starting from the base graph with $n = 0$ (no transitions), as in [2]_.
+    This is different from the parameter $t = n - 1$ in [1]_.
```

Comment:
Let's avoid the fraction latex feature for ease of reading (arguably on the html pages, but more so for the source code) and use `**` instead of `^`. Also any variables and simple expressions like n=0 should not be latex either.  We do support and use latex, but we try to use ascii when that is just as clear -- i.e. for simple expressions that don't use fancy mathematical notation.  I think we can safely write all of these two paragraphs without latex.

Reply:
I don't feel strongly either way, so whatever you feel is best works for me. Will do.